packages/sampleProp.conf

```diff
@@ -49,3 +49,9 @@ LivingRoomTemperatureTopic = "home/living-room/temperature"
 LivingRoomHumidityTopic = "home/living-room/humidity"
 LivingRoomLightTopic = "home/living-room/light"
 
+
+
+#### [CIP EthernetIP Parameters] ####
+
+CipEtherNetIp = "127.0.0.1"
+Tag = "Scada[0]"
```

Comment:
I think this should just be "Scada". The value from the tag is read at [0] position. That should be done by the program.

Reply:
I changed this.